lib/spack/spack/build_systems/xorg.py

```diff
@@ -6,12 +6,17 @@
 
 import spack.package_base
 import spack.util.url
+from spack.directives import conflicts
 
 
 class XorgPackage(spack.package_base.PackageBase):
     """Mixin that takes care of setting url and mirrors for x.org
     packages."""
 
+    # x.org packages are not supported on windows or darwin
+    conflicts("platform=windows")
+    conflicts("platform=darwin")
```

Comment:
Did you consider:
```
requires("platform=linux")
```
?

Reply:
Is freebsd still an option?

Reply:
Minor point though, feel free to merge like this if you like.

Reply:
I'll keep it as `conflicts`, since I am not sure how to add multiple options to a `requires` statement (`requires("platform=linux,freebsd")`?); it doesn't seem to be done yet for other packages in the builtin repository.